scipy/stats/_multivariate.py

```diff
@@ -305,6 +305,24 @@ class multivariate_normal_gen(multi_rv_generic):
     entropy()
         Compute the differential entropy of the multivariate normal.
 
+    Attributes
+    ----------
+    mean : ndarray
+        Mean of the distribution.
+        
+        .. versionadded:: 1.10.1
+        
+    cov : ndarray
+        Covariance matrix of the distribution.
+
+        .. versionadded:: 1.10.1
+
+    cov_object : `Covariance`
+        Representation of the covariance matrix as a `Covariance` object.
+
+        .. versionadded:: 1.10.1
+
+
```

Comment:
```suggestion
```
This is the documentation of `multivariate_normal_gen`, but `mean`, `cov`, and `cov_object` are only attributes of `multivariate_normal_frozen`. I don't think we want these changes to `multivariate_normal_gen`.